jobs/service-fabrik-postgresqlmt-operator/templates/config/settings.yml.erb

```diff
@@ -54,6 +72,24 @@ production:
     identity_provider: <%= identity_provider %>
     <% end %>
 
+  ###################
```

Comment:
This config was added as const { BaseService } = require('@sf/provisioner-services'); now loads DirectorService also. In the job PostgresqlMTOperator the configs for backup is not specified and this causes error when DirectorService also tries to load backup related modules.